test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -931,6 +1087,49 @@ TEST_P(ExtProcIntegrationTest, GetAndRespondImmediatelyWithBadStatus) {
   EXPECT_EQ("{\"reason\": \"Because\"}", response->body());
 }
 
+// EXTANT IN 1.22
+// These tests existed prior to the changes being backported from 1.22
```

Comment:
5ced3041fc2da22fcf72fd621ce432aa26024caa
signature change only; doesn't seem needed. will make reasonable effort to compile regardless